react/features/variant/V3C/content.jsx

```diff
@@ -0,0 +1,224 @@
+import React, { Component, useEffect, useState } from 'react';
+import Graph from './content/graph.jsx'
+import Info from './content/info.jsx'
+import Panel from './content/panel.jsx'
+import * as d3 from "d3";
+
+function Content (props) {
+
+    const chr = props.chr,
```

Comment:
In all components the props are first copied to new variables instead of using the props directly - is the reasoning that then it's clear in the beginning of the component what the props are or something else?

Reply:
Should we add Federico to this repo so that he could comment on this? I guess eventually he will do the V3C updates directly to this repository, if I understand it correctly.